GatewayAPI/GatewayAPI/ocelot.json

```diff
@@ -0,0 +1,487 @@
+{
+    "Routes": [
+        // InnoClinic.AuthorizationAPI (https://localhost:7238)
+        {
+            "UpstreamPathTemplate": "/gateway/auth/register",
```

Comment:
it's better to avoid 'gateway' word, so route will be just 'auth/register' here
remove it everywhere